Non-unity build failure

### Environment

Windows UE 5.1 using Incredibuild (non-unity builds for caching)

How do you use Sentry?
Sentry SaaS (sentry.io) or self-hosted/on-premise (which version?)

SaaS

Which version of the SDK?

0.16.0

How did you install the package? (Git-URL, Assetstore)

github

Which version of Unreal?

5.1.1 compiled from source

Is this happening in Unreal (editor) or on a player like Android, iOS, Windows?

### Steps to Reproduce

1. [Configure system to build non-unity, non-PCH](https://forums.unrealengine.com/t/how-to-compile-in-non-unity-mode/94863/3)
2. build project
3. Plugins\sentry-unreal-0.16.0-engine5.1-github\Source\SentryEditor\Private\SentrySymToolsDownloader.cpp(9): error C4668: 'ENGINE_MAJOR_VERSION' is not defined as a preprocessor macro, replacing with '0' for '#if/#elif'

### Expected Result

Build should be ok.

### Actual Result

What actually happened. Maybe a screenshot/recording? Maybe some logs?


### Any logs or screenshots


### Possible Solution
Add `#include "Runtime/Launch/Resources/Version.h"` to SentrySymToolsDownloader.cpp before the `ENGINE_MAJOR_VERSION` macro is used

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Fixes
+
+- Fix misssing include in non-unity builds ([#554](https://github.com/getsentry/sentry-unreal/pull/554))
+
 ### Dependencies
 
 - Bump CLI from v2.31.0 to v2.31.2 ([#551](https://github.com/getsentry/sentry-unreal/pull/551), [#553](https://github.com/getsentry/sentry-unreal/pull/553))

--- a/plugin-dev/Source/SentryEditor/Private/SentrySymToolsDownloader.cpp
+++ b/plugin-dev/Source/SentryEditor/Private/SentrySymToolsDownloader.cpp
@@ -2,6 +2,7 @@
 
 #include "SentrySymToolsDownloader.h"
 
+#include "Runtime/Launch/Resources/Version.h"
 #include "HttpModule.h"
 #include "Interfaces/IHttpRequest.h"
 #include "Interfaces/IHttpResponse.h"

